django/contrib/admin/static/admin/css/forms.css

```diff
@@ -184,7 +184,7 @@ form .aligned table p {
     width: auto;
     display: inline-block;
     vertical-align: -3px;
-    padding: 0 0 5px 5px;
+    padding: 1px 0 0 5px;
```

Comment:
Tested, this looks great! Also nice that this aligns better with how we handle this class in other css files 👍 https://github.com/django/django/blob/62300b81cf7687d176af3b40aa6101942801292d/django/contrib/admin/static/admin/css/responsive.css#L592

---
Tested a bit more and I think we can reduce this down to:
```css
.aligned .vCheckboxLabel {
    padding: 1px 0 0 5px;
}
```
This looks the same to me from my testing, do you agree?

Reply:
Yeah looks same to me I think we should go with this!